downloader-android/src/main/java/com/cloudinary/android/downloader/downloaders/fresco/FrescoDownloadRequestBuilderFactory.kt

```diff
@@ -0,0 +1,18 @@
+package com.cloudinary.android.downloader.downloaders.fresco
+
+import android.content.Context
+import com.cloudinary.Cloudinary
+import com.cloudinary.android.downloader.DownloadRequestBuilder
+import com.cloudinary.android.downloader.DownloadRequestBuilderFactory
+import com.cloudinary.android.downloader.downloaders.DownloadRequestBuilderImpl
+
+class FrescoDownloadRequestBuilderFactory : DownloadRequestBuilderFactory {
+
+    override fun createDownloadRequestBuilder(context: Context, cloudinary: Cloudinary): DownloadRequestBuilder {
+        return DownloadRequestBuilderImpl(
```

Comment:
Here too you can declare the function with a `=` and skip the `return` and returned type declaration.